apps/platforms/linux/gnome-terminal/gnome-terminal.py

```diff
@@ -42,7 +42,9 @@ class EditActions:
     def page_down(): actions.key('shift-pagedown')
     def page_up(): actions.key('shift-pageup')
     def paste(): actions.key('ctrl-shift-v')
-    def copy(): actions.key('ctrl-shift-c')
+    def copy(): 
+        actions.key('ctrl-shift-c')
```

Comment:
A `copy` action which takes the string representing keys to be pressed and calls both `keys()` and `clip_updated()` would likely make sense so that we don't need to hardcode `clip_updated()` in so many places.

Reply:
Then each app's `copy()` implementation can call `generic_copy()` with just the keys

Reply:
The latest version should address this concern indirectly, resolving